src/context_diff.rs

```diff
@@ -265,12 +265,33 @@ fn make_diff(
     results
 }
 
+fn get_modification_time(file_path: &str) -> String {
+    use chrono::{DateTime, Local};
+    use std::fs;
+
+    let metadata = fs::metadata(file_path).expect("Failed to get metadata");
```

Comment:
If, for whatever reason, the call to `fs::metadata(file_path)` returns an error, this is [going to panic](https://doc.rust-lang.org/std/io/type.Result.html#method.expect). Can you please handle the error case explicitly (ideally see how GNU diff handles cases where it cannot read an input file's metadata and implement something similar).

Reply:
Sure! I'm looking into it.

Reply:
I was not able to test it but I found this piece of code in GNU diff.
```c
static void
set_mtime_to_now (struct stat *st)
{
#ifdef STAT_TIMESPEC
  gettime (&STAT_TIMESPEC (st, st_mtim));
#else
  struct timespec t;
  gettime (&t);
  st->st_mtime = t.tv_sec;
# if defined STAT_TIMESPEC_NS
  STAT_TIMESPEC_NS (st, st_mtim) = t.tv_nsec;
# elif defined HAVE_STRUCT_STAT_ST_SPARE1
  st->st_spare1 = t.tv_nsec / 1000;
# endif
#endif
}
```

We should do the same, set modification time to `now`!

Reply:
Done!